content/en/docs/reference/command-line-tools-reference/feature-gates/service-account-token-pod-node-info.md

```diff
@@ -9,6 +9,9 @@ stages:
   - stage: alpha 
     defaultValue: false
     fromVersion: "1.29"
+  - stage: beta
+    defaultValue: true
+    fromVersion: "v1.30"
```

Comment:
Please consider adding the `toVersion` in the previous stage to align with consistency across other feature gate files. Additionally, the current `fromVersion` value (v1.30) is not in correct format, causing the PR [build to fail](https://app.netlify.com/sites/kubernetes-io-vnext-staging/deploys/65f18b217efc2a0008dfa949). Please consider the following suggestion:


```suggestion
    toVersion: "1.29"
  - stage: beta
    defaultValue: true
    fromVersion: "1.30"
```